isisd/isis_spf.c

```diff
@@ -1342,7 +1343,7 @@ static int isis_run_spf(struct isis_area *area, int level, int family,
 	isis_route_validate(area);
 	spftree->runcount++;
 	spftree->last_run_timestamp = time(NULL);
-	spftree->last_run_monotime = monotime(&time_now);
+	spftree->last_run_monotime = mnow;
```

Comment:
Doesn't this break end_time and then subsuquently spftree->last_run_duration?  I don't know enough about IS-IS to know if this matters or if this is just an accounting calculation.

Reply:
Yes, you are correct and I overlooked this. This breaks the algorithm runtime calculation. I'll fix this, but this shouldn't affect the ISIS functionality as that variable is just used by VTY to display the amount of time taken to process the SPF.